🐙 destination-cumulio: run up-to-date pipeline [2024-09-21]

airbyteio opened the pull request:
# Update destination-cumulio

This PR was autogenerated by running `airbyte-ci connectors --name=destination-cumulio up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Skipped

- Update versions of libraries in poetry.: Successful

- PATCH bump destination-cumulio version to 0.1.20: Successful

- Build destination-cumulio docker image for platform(s) linux/amd64, linux/arm64: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=45831) <br/>All committers have signed the CLA.

airbyteio committed:
Update versions of libraries in poetry.
PATCH bump destination-cumulio version to 0.1.20
Create or update pull request on Airbyte repository
Add changelog entry
